app/constants/commands.py

```diff
@@ -767,13 +787,11 @@ async def addnote(ctx: Context) -> Optional[str]:
     if not (t := await app.state.sessions.players.from_cache_or_sql(name=ctx.args[0])):
         return f'"{ctx.args[0]}" not found.'
 
-    log_msg = f'{ctx.player} added note: {" ".join(ctx.args[1:])}'
-
     await app.state.services.database.execute(
         "INSERT INTO logs "
-        "(`from`, `to`, `msg`, `time`) "
-        "VALUES (:from, :to, :msg, NOW())",
-        {"from": ctx.player.id, "to": t.id, "msg": log_msg},
+        "(`from`, `to`, `action`, `msg`, `time`) "
+        "VALUES (:from, :to, 'note', :msg, NOW())",
+        {"from": ctx.player.id, "to": t.id, "msg": " ".join(ctx.args[1:])},
```

Comment:
would prefer `:action` in the sql string, and add `"action": "note"` to the values here, more readable

Reply:
fixed in https://github.com/cmyui/gulag/commit/d905588385693702b0247e884b763a916beb40f6